docs/B-Computations/style-guidelines.md

```diff
@@ -1,5 +1,8 @@
 ---
 sidebar_position: 5
+title: "Style-Guidelines"
+id: "style-guidlines"
```

Comment:
```suggestion
id: style-guidelines
```
Typo.

Reply:
Thank you for the reference to #64. I have updated my code.